### ISSUE TITLE
InstancesTest.java:186-189: Create integration tests for Config....

### ISSUE THREAD

**Issue by dmarkov** [jcabi/jcabi-mysql-maven-plugin#58]
Puzzle `8-aea18ffe` in `src/test/java/com/jcabi/mysql/maven/plugin/InstancesTest.java:186-189` has to be resolved: Create integration tests for Config. Integration tests 'WithConfigITCase' should be created to test that user name, password and dbname are set properly. This issue should be done after non root user name is set properly

If you have any technical questions, don't ask me, submit new tickets instead

**Comment by dmarkov** [comment]
@apocarteres please proceed, it's yours

**Comment by apocarteres** [comment]
@dmarkov it's blocked by https://github.com/jcabi/jcabi-mysql-maven-plugin/issues/59

**Comment by apocarteres** [comment]
@dmarkov please find someone to review PR #61

**Comment by apocarteres** [comment]
@dmarkov it's impossible to go next here, since there is a bug in Travis build described in ticket #63

**Comment by yegor256** [comment]
@dmarkov we're blocked by #63

**Comment by dmarkov** [comment]
> @dmarkov we're blocked by #63

@yegor256 sure, we'll wait for #63

**Comment by apocarteres** [comment]
@dmarkov i'm sorry, but i can't wait more for response on https://github.com/jcabi/jcabi-maven-skin/issues/63 which blocks current issue. Please re-assign this task on someone else

**Comment by dmarkov** [comment]
> @dmarkov i'm sorry, but i can't wait more for response on https://github.com/jcabi/jcabi-maven-skin/issues/63 which blocks current issue. Please re-assign this task on someone else

@apocarteres all right, we'll find someone else for this task

**Comment by dmarkov** [comment]
@adko-pl please proceed, it's yours

**Comment by adko-pl** [comment]
@dmarkov please find another developer

**Comment by adko-pl** [comment]
@dmarkov @yegor256 please find another developer

**Comment by yegor256** [comment]
@dmarkov put someone else here pls

**Comment by dmarkov** [comment]
> @dmarkov @yegor256 please find another developer

@adko-pl -30 points to your rating :(

**Comment by dmarkov** [comment]
> @dmarkov @yegor256 please find another developer

@adko-pl I will ask somebody else to pick this up

**Comment by dmarkov** [comment]
> @dmarkov put someone else here pls

@yegor256 30 points deducted from your rating

**Comment by dmarkov** [comment]
> @dmarkov put someone else here pls

@yegor256 someone else will help in this task, no problem at all

**Comment by dmarkov** [comment]
@yamstranger the task is yours please proceed

**Comment by YamStranger** [comment]
@dmarkov I need more time here

**Comment by dmarkov** [comment]
> @dmarkov I need more time here

@yamstranger no problem, thanks for letting me know

**Comment by YamStranger** [comment]
@dmarkov i can't wait more for response on https://github.com/jcabi/jcabi-maven-skin/issues/63 please find another developer

**Comment by dmarkov** [comment]
> @dmarkov i can't wait more for response on https://github.com/jcabi/jcabi-maven-skin/issues/63 please find another developer

@yamstranger -30 points to your rating

**Comment by dmarkov** [comment]
> @dmarkov i can't wait more for response on https://github.com/jcabi/jcabi-maven-skin/issues/63 please find another developer

@yamstranger I will ask somebody else to pick this up

**Comment by dmarkov** [comment]
@tmjee the task is for you now, follow [these guidelines](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Don't hesitate to ask any technical questions right here

The cost of this task is **30 mins** (this is exactly how much will be paid, not less not more), when the task is [done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by tmjee** [comment]
@dmarkov  blocked by issue #78

**Comment by dmarkov** [comment]
> @dmarkov  blocked by issue #78 

@tmjee agreed, we'll wait for #78

**Comment by tmjee** [comment]
@dmarkov  Please assign to someone else. Please do not assign any more issues to me. Thanks.

**Comment by dmarkov** [comment]
> @dmarkov  Please assign to someone else. Please do not assign any more issues to me. Thanks.

@tmjee I deducted 30 points from your rating

**Comment by dmarkov** [comment]
> @dmarkov  Please assign to someone else. Please do not assign any more issues to me. Thanks.

@tmjee right, I'll try to find someone else for this task

**Comment by dmarkov** [comment]
@jrdalpra it's yours, please go ahead

**Comment by jrdalpra** [comment]
@dmarkov the issue #63  that was blocking @apocarteres was not solved. Must I proceed?

**Comment by jrdalpra** [comment]
@dmarkov This project is not stable. How can I start working if not even the master is working at CI server as you can see here https://ci.appveyor.com/project/yegor256/jcabi-mysql-maven-plugin and here https://travis-ci.org/jcabi/jcabi-mysql-maven-plugin.

Should I open and issue to solve problems related above? Thanks.

**Comment by jrdalpra** [comment]
@dmarkov can you see my comments above and give an answer? Thanks :)

**Comment by dmarkov** [comment]
> @dmarkov the issue #63  that was blocking @apocarteres was not solved. Must I proceed?

@jrdalpra it's up to you

**Comment by dmarkov** [comment]
> @dmarkov This project is not stable. How can I start working if not even the master is working at CI server as you can see here https://ci.appveyor.com/project/yegor256/jcabi-mysql-maven-plugin and here https://travis-ci.org/jcabi/jcabi-mysql-maven-plugin.
> 
> Should I open and issue to solve problems related above? Thanks.

@jrdalpra every time you see a problem, which is not related to the task you're working on, submit a new ticket, see http://www.yegor256.com/2015/01/15/how-to-cut-corners.html

**Comment by jrdalpra** [comment]
@dmarkov ok. So after I do that, should I wait or ask you to remove this issue from me?

**Comment by jrdalpra** [comment]
@dmarkov this issue is referenced at #78 
That's why I can't proceed.

**Comment by dmarkov** [comment]
> @dmarkov ok. So after I do that, should I wait or ask you to remove this issue from me?

@jrdalpra it's up to you. you can wait for your impediments to be resolved and continue here. or you can drop this task and lose points

**Comment by jrdalpra** [comment]
@dmarkov it's is not fully true. 

If I give up from this task I'll lose point.

If I wait I'll loose point too, because I won't be able to solve at time to get bonuses.

If the impediments take too long to be solved, I'll loose points too because the task will be taken from me after 10 days.

I think the best approach is to pause the counters of speed and rating until the impediments are solved. Then I can get this task back and solve it.

Other good way to solve it is to remove the task from me and remove me no point. That's not my fault that I can't proceed.

What do you think? 

P.S: I'll not giving up from the task :)

**Comment by dmarkov** [comment]
> @dmarkov it's is not fully true. 
> 
> If I give up from this task I'll lose point.
> 
> If I wait I'll loose point too, because I won't be able to solve at time to get bonuses.
> 
> If the impediments take too long to be solved, I'll loose points too because the task will be taken from me after 10 days.
> 
> I think the best approach is to pause the counters of speed and rating until the impediments are solved. Then I can get this task back and solve it.
> 
> Other good way to solve it is to remove the task from me and remove me no point. That's not my fault that I can't proceed.
> 
> What do you think? 
> 
> P.S: I'll not giving up from the task :)

@jrdalpra just ask for more time

**Comment by dmarkov** [comment]
@yegor256 impediment in issue #63 is closed, we can probably try to continue here

**Comment by jrdalpra** [comment]
@dmarkov give this task to someone else. I'm on vacation and can't do it. Thanks.

**Comment by dmarkov** [comment]
> @dmarkov give this task to someone else. I'm on vacation and can't do it. Thanks.

@jrdalpra deducted 30 from your rating :(

**Comment by dmarkov** [comment]
> @dmarkov give this task to someone else. I'm on vacation and can't do it. Thanks.

@jrdalpra OK, I will try to assign someone else

**Comment by dmarkov** [comment]
@yegor256 Thanks, closed
